### ISSUE TITLE
tiles: make it responsive to sidebars

### ISSUE THREAD

**Issue by merefield** [angusmcleod/discourse-topic-previews#71]
make tiles responsive to sidebars so they don't cause a cramped view with 3 columns squeezed together.

**Comment by merefield** [comment]
Almost there with this.  Have an experimental alternative tiles renderer with CS Grid working. Only issue is that Chrome based browsers cant cope with the large numbers of rows involved so may have to wait til that is resolved.

**Comment by angusmcleod** [comment]
```
so may have to wait til that is resolved.
```
Do you mean until Chrome is updated? You might be waiting a while.

**Comment by merefield** [comment]
Yes.  [Chrome only supports 1000 rows](https://bugs.chromium.org/p/chromium/issues/detail?id=688640), whereas Firefox supports 10,000 rows.   This limit imho prevents us from adopting this approach until that limit is lifted.  Of course there is a big risk it won't be as the grid was never supposed to support this many rows (tens of thousands) and presumably they are trying to minimise memory use.  However [CSS Grid 2.0 is coming](https://www.w3.org/TR/css-grid-2/) and apparently this may be dealt with as part of that.

**Comment by merefield** [comment]
This was implemented, but not relevant to the sidecar anyway.
